worker/pkg/workflows/datasync/activities/activities_test.go

```diff
@@ -206,6 +210,114 @@ output:
 	require.NoError(t, err)
 }
 
+type ExpectedJsValue struct {
+	Name string
+}
+
+func Test_Sync_Run_Success_Javascript(t *testing.T) {
+	testSuite := &testsuite.WorkflowTestSuite{}
+	env := testSuite.NewTestActivityEnvironment()
+
+	activities := &Activities{}
+	env.RegisterActivity(activities)
+
+	// benthos writes to stdout, need to capture that to evaluate the returned value, so we create a pipe to capture what is written to stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	val, err := env.ExecuteActivity(activities.Sync, &SyncRequest{
+		BenthosConfig: strings.TrimSpace(`
+input:
+  generate:
+    mapping: root = {"name":"evis"}
+    interval: 1s
+    count: 1
+pipeline:
+  processors:
+    - javascript:
+        code: |
+          (() => { 
+          function fn1(value){
+          var a = value + "test";
+          return a };
+          const input = benthos.v0_msg_as_structured();
+          const output = { ...input };
+          output["name"] = fn1(input["name"]);
+          benthos.v0_msg_set_structured(output);
+          })();
```

Comment:
We need a test here that shows what this looks like with multiple columns being transformed.

Also it would be great if this were formatted better.

Reply:
added a test to the benthos_builder_test file called `Test_ProcessorConfigMultiJavascript`